tests/test_job.py

```diff
@@ -1,7 +1,141 @@
-from pasqal_cloud import Job
+from datetime import datetime
+from typing import Any, Generator, List, Union
+from unittest.mock import patch
+from uuid import UUID, uuid4
+
+import pytest
+
+from pasqal_cloud import (
+    Job,
+    JobCancellingError,
+    JobFetchingError,
+    JobFilters,
+    PaginatedResponse,
+    PaginationParams,
+    SDK,
+)
+from pasqal_cloud.utils.constants import JobStatus
+from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
+from tests.utils import build_query_params
 
 
 class TestJob:
+    @pytest.fixture(autouse=True)
+    @patch(
+        "pasqal_cloud.client.Auth0TokenProvider",
+        FakeAuth0AuthenticationSuccess,
+    )
+    def _init_sdk(self):
```

Comment:
Doesn't need the `_` suffix

Reply:
I only applied the following rule of ruff: https://docs.astral.sh/ruff/rules/pytest-missing-fixture-name-underscore/

It raises an error during the pre-commit commit hook now as ruff is part of the dependencies. If you disagree with this rule I can change it.